doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>This is not rendering properly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>